src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -18,39 +18,46 @@
 @RegisterRowMapper(ConsentMapper.class)
 public interface ConsentDAO extends Transactional<ConsentDAO> {
 
-    @SqlQuery("select * from consents where consentId = :consentId and active=true")
+    /**
+     * Find consent by id
+     *
+     * @return Consent
+     */
+    @SqlQuery("SELECT * FROM consents WHERE consent_id = :consentId AND active=true")
     Consent findConsentById(@Bind("consentId") String consentId);
 
     @SqlQuery(
-        " SELECT c.* "
-            + " FROM consents c "
-            + " INNER JOIN consentassociations cs ON c.consentid = cs.consentid "
-            + " WHERE cs.datasetid = :datasetId")
+            " SELECT c.* "
+                    + " FROM consents c "
+                    + " INNER JOIN consent_associations ca ON c.consent_id = ca.consent_id "
+                    + " WHERE ca.dataset_id = :datasetId")
     Consent findConsentFromDatasetID(@Bind("datasetId") Integer datasetId);
 
     @SqlQuery("SELECT c.name " +
-            "FROM consents c INNER JOIN consentassociations cs ON c.consentId = cs.consentId "+
-            "WHERE cs.dataSetId = :datasetId")
+            "FROM consents c INNER JOIN consent_associations ca ON c.consent_id = ca.consent_id "+
+            "WHERE ca.dataset_id = :datasetId")
     String findConsentNameFromDatasetID(@Bind("datasetId") Integer datasetId);
 
-    @SqlQuery("select * from consents  where consentId in (<consentIds>)")
+    @SqlQuery("SELECT * FROM consents WHERE consent_id IN (<consentIds>)")
     Collection<Consent> findConsentsFromConsentsIDs(@BindList("consentIds") List<String> consentIds);
 
-    @SqlQuery("select * from consents  where name in (<names>)")
+    @SqlQuery("SELECT * FROM consents WHERE name IN (<names>)")
     List<Consent> findConsentsFromConsentNames(@BindList("names") List<String> names);
```

Comment:
This has 1 usage in a test ... we can remove this.

Reply:
Removed **findConsentsFromConsentNames** and the associated test.